<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>304</x>
        <y>93</y>
        <width>123</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Host Lobby</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>304</x>
        <y>180</y>
        <width>123</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Join lobby</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chatRoom">
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>6</y>
        <width>788</width>
        <height>547</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>557</y>
        <width>791</width>
        <height>38</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="connect">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>327</x>
        <y>337</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ip">
      <property name="geometry">
       <rect>
        <x>327</x>
        <y>220</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>268</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>8080</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>244</y>
        <width>104</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>328</x>
        <y>198</y>
        <width>104</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ip</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>334</x>
        <y>142</y>
        <width>104</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>329</x>
        <y>170</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>username</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
